monai/transforms/utility/dictionary.py

```diff
@@ -1740,6 +1741,69 @@ def __call__(self, data: Mapping[Hashable, NdarrayOrTensor]) -> dict[Hashable, N
         return d
 
 
+class ApplyTransformToPointsd(MapTransform, InvertibleTransform):
+    """
+    Dictionary-based wrapper of :py:class:`monai.transforms.ApplyTransformToPoints`.
+    The input coordinates are assumed to be in the shape (C, N, 2 or 3),
+    where C represents the number of channels and N denotes the number of points.
+
+    Args:
+        keys: keys of the corresponding items to be transformed.
+            See also: monai.transforms.MapTransform
+        refer_key: The key of the reference item used for transformation.
+            It can directly refer to an affine or an image from which the affine can be derived.
+        dtype: The desired data type for the output.
+        affine: A 3x3 or 4x4 affine transformation matrix applied to points. This matrix typically originates
```

Comment:
In `ApplyTransformToPoints`, the `affine` parameter is optional, but it's not clear what happens when it's `None`. It would be good to add a docstring explaining the behavior in this case.

Reply:
If the parameter is `None`, does it not use the affine matrix from the image ? I believe that should be expected behavior, right ?

Reply:
Yes, you are right @vikashg. Add more description.